generated/json_data/qol.jsonc

```diff
@@ -4590,6 +4590,26 @@
                     ]
                 },
                 "Sun Tower Access": {
+                    "deleteIds": [
```

Comment:
I don't understand why you are moving these to qol from skippable. I thought the crash risk was introduced by skippable because it's adding several objects to those rooms. If that's true, the crash fix needs to stay with the crash risk.